### PR TITLE
Add `1.112.0` changelog

### PR DESCRIPTION
**Pull request by confused-Techie**
This PR adds the changelog, and changelog notes to the `welcome` package.

Relates to #834

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,30 @@
 
 ## [Unreleased]
 
+## 1.112.0
+
+- Added the ability for a user to indicate when to automatically show or hide the wrap-guide; between always, when soft wrap is enabled, or when soft wrap at preferred line length is enabled.
+- Updated network handling in PPM to something newer and more secure.
+- Made PPM async.
```
Comment (DeeDeeG) [inline CHANGELOG.md:13]:
```suggestion
- Updated most of PPM's code to use async/await and promises internally.
```

File: packages/welcome/lib/changelog-view.js [hunk 2]
```diff
@@ -50,22 +50,19 @@ export default class ChangeLogView {
...
-                Added a new "UI" API to `atom`, accessible via `atom.ui`. This exposes a `markdown` object, allowing community packages to offload Markdown handling to the core editor.
+                Added the ability for a user to indicate when to automatically show or hide the wrap-guide; between always, when soft wrap is enabled, or when soft wrap at preferred line length is enabled.
               </li>
               <li>
-                Fine-tuned/deduped dependencies to remove ~35.5 MB from Pulsar's installed size.
+                Updated network handling in PPM to something newer and more secure.
               </li>
               <li>
-                Fixed an issue that sometimes caused text to shift or disappear after an editor pane regains focus.
+                Made PPM async.
```
Comment (DeeDeeG) [inline packages/welcome/lib/changelog-view.js:59]:
```suggestion
                Updated most of PPM's code to use async/await and promises internally.
```

**Review by DeeDeeG — COMMENTED** [review]
Suggesting an alternate ppm "asyncify" release note blurb line.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,30 @@
 
 ## [Unreleased]
 
+## 1.112.0
+
+- Added the ability for a user to indicate when to automatically show or hide the wrap-guide; between always, when soft wrap is enabled, or when soft wrap at preferred line length is enabled.
```
Comment (Daeraxa) [inline CHANGELOG.md:11]:
```suggestion
- Added options for a user to control when to automatically show or hide the wrap-guide; "Always", "When soft wrap is enabled", and "When soft wrap at preferred line length is enabled".
```
Suggestion to quote the the actual descriptions from the option and removes ambiguity from the "when"s as it reads like part of a sentence.

File: packages/welcome/lib/changelog-view.js [hunk 4]
```diff
@@ -50,22 +50,19 @@ export default class ChangeLogView {
             <p>Feel free to read our <a href="https://github.com/pulsar-edit/pulsar/blob/master/CHANGELOG.md">Full Change Log</a>.</p>
             <ul>
               <li>
-                Added a new "UI" API to `atom`, accessible via `atom.ui`. This exposes a `markdown` object, allowing community packages to offload Markdown handling to the core editor.
+                Added the ability for a user to indicate when to automatically show or hide the wrap-guide; between always, when soft wrap is enabled, or when soft wrap at preferred line length is enabled.
```
Comment (Daeraxa) [inline packages/welcome/lib/changelog-view.js:53]:
```suggestion
                Added the ability for a user to indicate when to automatically show or hide the wrap-guide; between always, when soft wrap is enabled, or when soft wrap at preferred line length is enabled.
```
                Added options for a user to control when to automatically show or hide the wrap-guide; "Always", "When soft wrap is enabled", and "When soft wrap at preferred line length is enabled".

**Review by DeeDeeG — APPROVED** [review]
Noice 👍 

(Thanks for putting this all together, @confused-Techie !)

**Pull request merged**